pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[build-system]
+requires = ["setuptools>=64.0.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "stylia"
+dynamic = ["version"]
+authors = [
+    {name = "Miquel Duran-Frigola, Ersilia Open Source Initiative", email = "miquel@ersilia.io"},
+]
+description = "Helper functions used by Ersilia to produce presentable plots with MatPlotLib"
+readme = "README.md"
+license = {text = "MIT"}
+requires-python = ">=3.7"
+dependencies = [
+    "seaborn",
+    "matplotlib",
+    "scikit-learn",
+    "cmcrameri",
+    "palettable"
+]

```

Comment:
Can we try to pin dependency versions?

Reply:
thanks, I've pinned the versions to stable ones...